Tools/Replay/Replay.cpp

```diff
@@ -932,4 +932,7 @@ const struct AP_Param::GroupInfo        GCS_MAVLINK::var_info[] = {
 };
 GCS_Dummy _gcs;
 
+// avoid linking against AP_AdvancedFailsafe
+bool AP_AdvancedFailsafe::gcs_terminate(bool should_terminate) {}
```

Comment:
Fixed, that was quite bad... I'm surprised the compiler didn't refuse to compile that.